### PR TITLE
Links to model source papers

### PR DESCRIPTION
**Pull request by keiran-rowell-unsw**

### CODE REVIEW

File: form.yml.erb [hunk 1]
```diff
@@ -26,10 +26,10 @@ attributes:
         label: "Method"
         help: |
             <ul>
-                <li><b>AlphaFold2.3</b> High Accuracy, Slower</li>
-                <li><b>ESMFold</b> Medium/Low Accuracy, Fastest, No Evolutionary Sequence Calculations</li>
-                <li><b>RoseTTAFold-All-Atom</b> High Accuracy, Slower</li>
-                <li><b>HelixFold3</b> High Accuracy, Slower</li>
+                <li><b>AlphaFold2.3</b> High Accuracy, Slower</li> - [Paper](https://doi.org/10.1038/s41586-021-03819-2)
```
Comment (nbtm-sh) [inline form.yml.erb:29]: Links should be within the `<li></li>` tags.

Comment (nbtm-sh) [inline form.yml.erb:29]: Links should be formatted using HTML, testing suggests markdown is not supported here
`<a href="https://doi.org/10.1038/s41586-021-03819-2">Paper</a>`

**Review by nbtm-sh — CHANGES_REQUESTED** [review]
Just need to format links using HTML and place them inside the list item tags.

File: form.yml.erb (hunk 1 above)
Comment (keiran-rowell-unsw) [reply form.yml.erb:29]: Aha! Will do in the moment. Thought I read OoD supported Markdown but maybe not in this field

Comment (nbtm-sh) [reply form.yml.erb:29]: I believe it is supported in some fields but here we should stick with HTML.

**Pull request closed without merging**